### DISCUSSION TITLE
Variablen für Farben

### DISCUSSION THREAD

**Discussion by AFPass** [Zefau/ioBroker.jarvis#1301] [Q&A]
Gibt es eine Möglichkeit für die  Line Color auch Variablen zu benutzen statt immer die Farbcodes?

**Comment by Zefau** [comment]
Du kannst fast überall CSS Variablen nutzen meine ich.

**Comment by AFPass** [comment]
Ich meine hier:
![grafik](https://user-images.githubusercontent.com/13101074/146152033-740742f1-3360-41f7-8c3d-ccbce45750f0.png)

**Reply by Zefau** [reply]
Hast du eine CSS Variable probiert?

```
var(--q-color-primary)
```

**Comment by AFPass** [comment]
Du meinst so?
![grafik](https://user-images.githubusercontent.com/13101074/146359105-f92399fc-0101-4171-928b-49966166a22d.png)

![grafik](https://user-images.githubusercontent.com/13101074/146359329-80ea95ef-d6c7-4561-a3f8-b6548f22bbd4.png)

Geht nicht.

**Reply by Zefau** [reply]
Jo so meinte ich

**Reply by Zefau** [reply]
Im Frontend greift es auch nicht?

**Comment by AFPass** [comment]
Leider nein
